### ISSUE TITLE
Duplicated locales (after nstallation)

### ISSUE THREAD

**Issue by bakbuz** [nopSolutions/nopCommerce#6495]
nopCommerce version: 4.60

Steps to reproduce the problem: 

Sql query:
```
select ResourceName, count(*) from LocaleStringResource
where LanguageId=1
group by ResourceName
having count(*)>1
```

Results:
![Capture](https://user-images.githubusercontent.com/2492378/209777173-3e5d3309-d29a-4231-9bc3-ba233dbf59fa.PNG)

**Comment by AndreiMaz** [comment]
We'll apply the fix soon. In the meantime, please find the SQL scripts attached. Execute one of them (for yourdatabase type) to delete the duplicated records:
[mssql_locale_string_resource_migrate.txt](https://github.com/nopSolutions/nopCommerce/files/10323791/mssql_locale_string_resource_migrate.txt)
[mysql_locale_string_resource_migrate.txt](https://github.com/nopSolutions/nopCommerce/files/10323792/mysql_locale_string_resource_migrate.txt)
[postgres_locale_string_resource_migrate.txt](https://github.com/nopSolutions/nopCommerce/files/10323793/postgres_locale_string_resource_migrate.txt)

**Comment by skoshelev** [comment]
Closed #6495
